src/test/java/teammates/test/cases/webapi/RestoreCourseActionTest.java

```diff
@@ -37,15 +37,14 @@ public void testExecute() throws Exception {
         ______TS("Typical case, restore a soft-deleted course from Recycle Bin");
```

Comment:
You should add more tests here?

- Not enough parameter?
- Non-existent?
- Not in recycle bin?

Reply:
Okay added 👍